src/workerd/server/tests/python/dont-snapshot-pyodide/worker.py

```diff
@@ -0,0 +1,2 @@
+async def test():
```

Comment:
How does this test the issue?

Reply:
To trigger the bug we need to do two things:
1. import `pyodide` at top level
2. ensure that there is some package requirement

The `numpy` import here is unnecessary, but if someone removed `(name = "numpy", pythonRequirement = "")` from the `wd-test` file it would no longer reproduce the problem.

Reply:
Oh shoot ruff removed the imports. I keep having this problem.

Reply:
But where is the pyodide import? 😅

Reply:
> Oh shoot ruff removed the imports. I keep having this problem.

Hahaha that explains it
There’s probably some comments that disable ruff on a file right?

Reply:
I put a whole explanation in now. Also #3415 is such a huge improvement to our test suite I am so happy we merged it.